deviceplugin/path.go

```diff
@@ -0,0 +1,114 @@
+// Copyright YEAR the generic-device-plugin authors
```

Comment:
CI might be complaining about this too
```suggestion
// Copyright 2023 the generic-device-plugin authors
```

Reply:
Resolved 3927c05